docs/howto/login-gdm.md

```diff
@@ -16,13 +16,13 @@ If MFA is enabled, a QR code and a login code are displayed.
 
 ![Display of QR code, login code and button to Request new login code.](../assets/gdm-qr.png)
 
-From a second device, flash the QR code or type the URL in a web browser, then follow the authentication process from your provider.
+From a second device, flash the QR code or type the URL in a web browser, then follow your provider's authentication process.
 
 Upon successful authentication, the user is prompted to enter a local password. This password can be used for offline authentication.
 
 ![Prompt to create local password on successful authentication.](../assets/gdm-pass.png)
 
-## Groups management
+## Group management
```

Comment:
I think here the plural form still stands, since we can manage _multiple groups_ with authd.

Reply:
Hmmmm... I had to think about this.

If we were referring to users here, we would still say "user management", right? Even if the scope includes multiple potential users. "Users management" reads awkwardly, and "Groups management" might do as well.

It's as if we are referring to the general type of entity being managed, rather than a single instance of that entity.

In Entra ID docs, the term "group management" is used: [example](https://learn.microsoft.com/en-us/entra/identity/users/groups-self-service-management) .

I would tend to keep the change ("Group management") or else change to "Managing groups", to avoid the awkward phrase.

Reply:
That's my reasoning, @edibotopic, we don't say "Users management" or "Projects management"; similarly it sounds odd to me to say "Groups management". As far as I understand, in English when a noun is used as an attribute it's typically used in the singular form.

Reply:
> That's my reasoning, @edibotopic, we don't say "Users management" or "Projects management"; similarly it sounds odd to me to say "Groups management". As far as I understand, in English when a noun is used as an attribute it's typically used in the singular form.

+1, "Groups management" feels wrong